syntax = "proto3";
package gdmp;

message Packet {
  int32 packet_type = 1;

  oneof packet {
    Placeholder placeholder   = 2;
    Placeholder2 placeholder2 = 3;
    Placeholder3 placeholder3 = 4;
  };
}

message RoomInfo {
  int32 level_id = 1;
  /* maybe more stuff, private rooms maybe? */
}

// --------- PACKET TYPES ----------
// these might be used for proper auth later
message Placeholder {}
message Placeholder2 {}
message Placeholder3 {}

message PlayerJoin {
    RoomInfo room = 1;
}

